Parse/PFSubclassing.h

```diff
@@ -36,13 +36,13 @@ NS_ASSUME_NONNULL_BEGIN
  Constructs an object of the most specific class known to implement `+parseClassName`.
 
  This method takes care to help `PFObject` subclasses be subclassed themselves.
- For example, `PFUser.+object` returns a `PFUser` by default but will return an
+ For example, `PFUser.+instantiateObject` returns a `PFUser` by default but will return an
  object of a registered subclass instead if one is known.
  A default implementation is provided by `PFObject` which should always be sufficient.
 
  @return Returns the object that is instantiated.
  */
-+ (instancetype)object;
++ (instancetype)instantiateObject;
```

Comment:
so let's make it private no? it's handled in PFObject.h right but not exposed publicly*